aspnetcore/data/ef-mvc/intro.md

```diff
@@ -220,15 +213,21 @@ In the *Data* folder, create a new class file named *DbInitializer.cs* and repla
 
 The code checks if there are any students in the database, and if not, it assumes the database is new and needs to be seeded with test data.  It loads test data into arrays rather than `List<T>` collections to optimize performance.
 
-In *Startup.cs*, modify the `Configure` method to call this seed method on application startup. First, add the context to the method signature so that ASP.NET dependency injection can provide it to your `DbInitializer` class.
+In *Program.cs*, modify the `Main` method to do the following on application startup:
+
+* Get a database context instance from the dependency injection container.
+* Call the seed method, passing to it the context.
+* Dispose the context when the seed method is done.
 
-[!code-csharp[Main](intro/samples/cu/Startup.cs?name=snippet_ConfigureSignature&highlight=1)]
+[!code-csharp[Main](intro/samples/cu/Program.cs?name=snippet_Seed&highlight=3-20)]
 
-Then call your `DbInitializer.Initialize` method at the end of the `Configure` method.
+Add `using` statements:
 
-[!code-csharp[Main](intro/samples/cu/Startup.cs?name=snippet_RouteAndSeed&highlight=8)]
+[!code-csharp[Main](intro/samples/cu/Program.cs?name=snippet_Usings)]
 
-Now the first time you run the application the database will be created and seeded with test data. Whenever you change your data model, you can delete the database, update your seed method, and start afresh with a new database the same way. In later tutorials you'll see how to modify the database when the data model changes, without deleting and re-creating it.
+Now the first time you run the application, the database will be created and seeded with test data. Whenever you change your data model, you can delete the database, update your seed method, and start afresh with a new database the same way. In later tutorials you'll see how to modify the database when the data model changes, without deleting and re-creating it.
```

Comment:
When the data model changes, delete the db, update the seed method. When the app starts new data is seeded.

On this section in the MVC tutorial I have lots of questions on why the seed method isn't running. It's almost always because they're running non-debug and the IIS Express is still running.  You need to exit IIS Express to force startup to run again. Do you ever get that question?